documentation/reference/document-json-update-format.html

```diff
@@ -74,90 +89,205 @@ <h3 id="assign">assign</h3>
     }
 }
 </pre>
-<code>assign</code> can use fieldpath syntax when used on composite types when they are not part of attribute or index.
-On a <code>map</code> it can be used as a put.
-In the following example a tuple (key1, value1) is inserted into the map
-regardless if there is an element with key "key1" before:
+<h3 id="assign-struct-field">Struct field</h3>
+<h5>Replacing all fields in a struct</h5>
+<p>
+Entire struct fields may be replaced by assigning a JSON object of struct key/value pairs.
+</p>
+<pre>
+struct person {
+  field first_name type string {}
+  field last_name type string {}
+}
+field contact type person {
+  indexing: summary
+}
+</pre>
 <pre>
 {
-    "update": "id:music:music::example2",
+    "update": "id:mynamespace:workers::example",
```

Comment:
yes! good practice to say mynamespace wherever, users are confused about music:music :-)

Reply:
I still get confused myself, to be honest 😬 Principle of Least Surprise™️ reigns supreme